x-pack/legacy/plugins/apm/server/routes/create_api.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { merge } from 'lodash';
+import Boom from 'boom';
+import { InternalCoreSetup } from 'src/core/server';
+import { Request, ResponseToolkit } from 'hapi';
+import * as t from 'io-ts';
+import { PathReporter } from 'io-ts/lib/PathReporter';
+import {
+  ServerAPI,
+  RouteFactoryFn,
+  HttpMethod,
+  Route,
+  Params
+} from './typings';
+
+export function createApi() {
+  const factoryFns: Array<RouteFactoryFn<any, any, any, any>> = [];
+  const api: ServerAPI<{}> = {
+    _S: {},
```

Comment:
Is `_S` something you came up with or where does it come from?

Reply:
Okay, thanks for the explanation. `_RouteState_ ` would be a bit more clear but I'm fine with either.